figures/sites.py

```diff
@@ -304,6 +305,24 @@ def _get_all_sites():
     return Site.objects.all()
 
 
+def get_requested_site(request):
+    """
+    From a request return the corresponding site.
+
+    This functions makes use of the `REQUESTED_SITE_BACKEND` setting if configured, otherwise
+    it defaults to Django's get_current_site().
+
```

Comment:
@OmarIthawi Please add  "Using the backend" instructions either here, in this module docstring or in a readme doc

Reply:
Thanks John! I'll add the instructions to the README doc.

Reply:
I've documented the feature in a new Configurations section. I'll merge after someone spot check the section.

It's not a complete one, but a start.